tensorflow/compiler/mlir/lite/transforms/prepare_tf.cc

```diff
@@ -38,17 +38,17 @@ limitations under the License.
 #include "llvm/ADT/StringSwitch.h"
 #include "llvm/Support/Casting.h"
 #include "llvm/Support/Debug.h"
-#include "mlir/Analysis/LoopAnalysis.h"  // from @llvm-project
```

Comment:
It looks like the PR needs to be rebased because I see a lot of unrelated changes

Reply:
What code formatting style is used in this repository? [This page](https://www.tensorflow.org/community/contribute/code_style#c_coding_style) leads me to believe that its just the plain google style from clang-format, but when I apply this style to files it results in all of these spurious changes (my fault for blanket applying it to the entire repository, I guess)

Reply:
Please only format the files you touch (I'm also surprised that there are such large diffs against clang-format; we generally disallow submitting changes that do not round-trip through our internal clang-format configuration)